src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -16,44 +16,71 @@ UI Options does three things:
    for deployment purposes).
 2. Unzip the contents of the downloaded ZIP file to a location within your project. This will result in a new directory
    `infusion`.<div class="infusion-docs-note"><strong>Note:</strong> In this guide we will use the directory
    `my-project/lib/`.</div>
-3. Your `infusion` folder will include a single file containing all of the JavaScript you need
+3. Your `infusion` directory will include a single file containing all of the JavaScript you need
    (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
    link to these files in your HTML files.
 4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-2.0.0.zip` (or similar name)
    from your download directory.
 
+### Github Source Method
+This method will download and install the most recent development version of Infusion.
+
+1. Clone the Infusion Github repository anywhere to your local filesystem: `git clone https://github.com/fluid-project/infusion`. This will create a directory `infusion` containing the Infusion source code.
+2. Within the `infusion` directory, run `npm install` to get Infusion's dependencies.
+3. Build a version of Infusion with UI Options by running `grunt custom --include="uiOptions" --name="uiOptions"`. This will create a build of Infusion in the directory `infusion/build/`. Note: [Get Grunt](https://gruntjs.com/) if you don't have it installed.
+4. Your `infusion/build` directory will include a single file containing all of the JavaScript you need
+   (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
+   link to these files in your HTML files.
+5. Rename the directory from `build` to `infusion`, and move (or copy) the `build` directory to your project (i.e. `my-project/lib/infusion`).
+6. Optional: You can now safely delete the `infusion` directory created in step 1.
+
+### npm Method
+This method will download and install the last release of Infusion (2.0.0).
+
+1. From your project directory, run `npm i infusion`.
+2. Within the `node_modules/infusion/` directory in your project, run `npm install` to get Infusion's dependencies.
+3. Then build a version of Infusion with UI Options by running `grunt custom --include="uiOptions" --name="uiOptions"`. This will create a build of Infusion in the `node_modules/infusion/build/` directory. Note: [Get Grunt](https://gruntjs.com/) if you don't have it installed.
+4. Your `node_modules/infusion/build` folder will include a single file containing all of the JavaScript you need
+   (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
+   link to these files in your HTML files.
+5. Rename the directory from `build` to `infusion`, and move (or copy) the `build` directory to your project (i.e. `my-project/lib/infusion`).
```

Comment:
Don't remember what was the case for the 2.0.0 release, but now at least you don't need to do all this work. The npm package includes a dist directory that includes builds of infusion. There's some copying around you'll have to do to move the files where you want them though, but you don't need to run your own builds.

Reply:
The dist directory includes different builds, but no UIO specific. I suppose I can mention they can use one of these builds, but if they want one with just UIO, then we can do a custom build.

- infusion-all-no-jquery.js
- infusion-all.js
- infusion-framework-no-jquery.js
- infusion-framework.js

Reply:
The dev releases include UIO specific builds. You can see from this one on unpkg ( https://unpkg.com/browse/infusion@3.0.0-dev.20190905T163833Z.b024bff87/dist/ ). So the next release (Infusion 3.0.0) would have it as well. Are we missing any other builds that we should be generating?

Reply:
In reading the documentation and the markup again, I don't think we should be recommending that users use `npm install` to then generate a custom build using grunt. I think we should expect that if someone is using npm to source Infusion that they are already using npm for their own project. In this case they should be using `npm install infusion --save` or `npm install infusion --save-dev` and performing some copy steps, either manual or automated through grunt or other means. 

If they want to do a custom build they should pull down the source code, either by git clone or getting a GitHub generated zip file of the repo.